{"sections":[],"hierarchy":{"paths":[["doc:\/\/com.auth0.Auth0\/documentation\/Auth0","doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError"]]},"schemaVersion":{"patch":0,"major":0,"minor":3},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError\/LocalizedError-Implementations"},"variants":[{"paths":["\/documentation\/auth0\/authenticationerror\/localizederror-implementations"],"traits":[{"interfaceLanguage":"swift"}]}],"topicSections":[{"generated":true,"title":"Instance Properties","identifiers":["doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError\/failureReason","doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError\/helpAnchor","doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError\/recoverySuggestion"]}],"kind":"article","metadata":{"modules":[{"name":"Auth0"}],"title":"LocalizedError Implementations","role":"collectionGroup"},"references":{"doc://com.auth0.Auth0/documentation/Auth0/AuthenticationError/helpAnchor":{"fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"helpAnchor","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"kind":"text","text":"?"}],"identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError\/helpAnchor","abstract":[],"title":"helpAnchor","type":"topic","url":"\/documentation\/auth0\/authenticationerror\/helpanchor","kind":"symbol","role":"symbol"},"doc://com.auth0.Auth0/documentation/Auth0/AuthenticationError/failureReason":{"fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"failureReason","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":"?","kind":"text"}],"identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError\/failureReason","abstract":[],"type":"topic","title":"failureReason","url":"\/documentation\/auth0\/authenticationerror\/failurereason","kind":"symbol","role":"symbol"},"doc://com.auth0.Auth0/documentation/Auth0":{"type":"topic","title":"Auth0","url":"\/documentation\/auth0","abstract":[{"type":"text","text":"SDK for Apple platforms."}],"identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0","role":"collection","kind":"symbol"},"doc://com.auth0.Auth0/documentation/Auth0/AuthenticationError/recoverySuggestion":{"fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"recoverySuggestion","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"kind":"text","text":"?"}],"identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError\/recoverySuggestion","abstract":[],"title":"recoverySuggestion","type":"topic","url":"\/documentation\/auth0\/authenticationerror\/recoverysuggestion","kind":"symbol","role":"symbol"},"doc://com.auth0.Auth0/documentation/Auth0/AuthenticationError":{"role":"symbol","fragments":[{"kind":"keyword","text":"struct"},{"text":" ","kind":"text"},{"text":"AuthenticationError","kind":"identifier"}],"title":"AuthenticationError","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError","kind":"symbol","url":"\/documentation\/auth0\/authenticationerror","type":"topic","abstract":[{"type":"text","text":"Represents an error during a request to the Auth0 Authentication API."}],"navigatorTitle":[{"kind":"identifier","text":"AuthenticationError"}]}}}